src/lib.rs

```diff
@@ -1,35 +1,37 @@
 extern "C" {
-    #![allow(improper_ctypes)] // we do not actually cross the FFI bound here
+#![allow(improper_ctypes)] // we do not actually cross the FFI bound here
```

Comment:
This… feels wrong.

Reply:
yeah you're right... that's cargofmt's fault. I moved things around so it's prettier.